recipe/build.sh

```diff
@@ -3,6 +3,8 @@
 set -ex
 
 export CXXFLAGS="${CXXFLAGS} -D__STDC_FORMAT_MACROS -D_LIBCPP_DISABLE_AVAILABILITY"
+export EMSDK_PATH=${CONDA_EMSDK_DIR}
+export WASI_SDK_PATH="" # FIXME
```

Comment:
```console
/bin/sh: /bin/clang: No such file or directory
make[2]: *** [src/runtime/legacy/CMakeFiles/lfortran_runtime_wasm.dir/build.make:73: src/runtime/legacy/lfortran_runtime_wasm.o] Error 127
make[1]: *** [CMakeFiles/Makefile2:692: src/run
```

We need the `WASI_SDK_PATH` to be set for the above. We use `WASI_SDK_PATH/bin/clang` to compile our runtime library `src/libasr/runtime/lfortran_intrinsics.c` to WebAssembly Object file which we can later link.